src/ConnectedMode/Binding/IUnintrusiveBindingController.cs

```diff
@@ -67,6 +70,17 @@ public async Task BindAsync(BoundServerProject project, IProgress<FixedStepsProg
             await bindingProcess.SaveServerExclusionsAsync(token);
         }
 
+        public bool Unbind(string localBindingKey)
+        {
+            var bindingDeleted = solutionBindingRepository.DeleteBinding(localBindingKey);
+            if (bindingDeleted)
+            {
+                sonarQubeService.Disconnect();
```

Comment:
https://github.com/SonarSource/sonarlint-visualstudio/blob/e0afc290d353c61961bf1fa9e5756372aa3c2ad6/src/Integration/MefServices/ActiveSolutionBoundTracker.cs#L119 <- this method can already handle the disconnect. I get why you're not using it, but maybe we need to generalize that for both solution change and unbind. Also it has a call to `gitEventsMonitor.Refresh();`, not sure if we need to call that here too (or in the new method)

Reply:
This follows the current pattern of managing the sonar qube client in the controller, so I'm resolving this for now